wiki/Bancho_(server)/en.md

```diff
@@ -1,11 +1,26 @@
 ---
-stub: true
 tags:
   - infrastructure
 ---
 
-# Bancho (game server)
+# Bancho (server)
 
 *For other uses, see [Bancho (disambiguation)](/wiki/Disambiguation/Bancho).*
 
-**osu!bancho** is the server which handles user presence, chat and multiplayer services.
+**Bancho** (previously *osu!bancho*) is a code name of the server component which handles interactions between players running stable versions of osu!:
+
+- [Online presence](/wiki/Client/Interface/Chat_console#extended-chat-console) and the [IRC gateway for chat](/wiki/Community/Internet_Relay_Chat)
+- [Spectator](/wiki/Gameplay/Spectating) mode
+- [Multiplayer system](/wiki/Client/Interface/Multiplayer) and [tournament management](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands)
```

Comment:
Probably mention chat? Not just the "IRC gateway". 
It's the number one primary purpose of bancho after all.

Reply:
added to the first item